CDCgov/prime-reportstream
# Added embedded example HL7 file. Placed example HL7 text into text bo…

Added embedded example HL7 file. Placed example HL7 text into text box to prevent word adding linebreaks and carriage returns where they should not be.

Signed-off-by: James.Gilmore-A <james.gilmore@agile6.com>

This PR ...

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. Copy/paste HL7 text from programmers guide into new .hl7 file
2. Test file with ReportStream Validation service and ensure it passes
3. Repeat step 2 with embedded HL7 file.

## Changes
-Update to ReportStream programmer's guide to f
[ReportStream-Programmers-Guide-v2.3.docx](https://github.com/CDCgov/prime-reportstream/files/9397657/ReportStream-Programmers-Guide-v2.3.docx)
ix Hl7 example text so that it can pass the ReportStream validation service.
-Add embedded Hl7 example file to programmer's guide.

## Checklist

### Testing
- [ x] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?

### Process

## Linked Issues
- Fixes  

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue 

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
